pom.xml

```diff
@@ -899,88 +900,6 @@
-                  <artifactId>hadoop-common</artifactId>
-                </exclusion>
-                <exclusion>
-                  <groupId>org.apache.hadoop</groupId>
-                  <artifactId>hadoop-hdfs</artifactId>
-                </exclusion>
-                <exclusion>
-                  <groupId>org.apache.hive</groupId>
-                  <artifactId>hive-storage-api</artifactId>
-                </exclusion>
-              </exclusions>
-            </dependency>
-            <dependency>
-              <groupId>org.apache.orc</groupId>
-              <artifactId>orc-mapreduce</artifactId>
-              <version>${orc.version}</version>
-              <classifier>${orc.classifier}</classifier>
-              <exclusions>
-                <exclusion>
-                  <groupId>org.apache.hadoop</groupId>
-                  <artifactId>hadoop-common</artifactId>
-                </exclusion>
-                <exclusion>
-                  <groupId>org.apache.hadoop</groupId>
-                  <artifactId>hadoop-mapreduce-client-core</artifactId>
-                </exclusion>
-                <exclusion>
-                  <groupId>org.apache.orc</groupId>
-                  <artifactId>orc-core</artifactId>
-                </exclusion>
-                <exclusion>
-                  <groupId>org.apache.hive</groupId>
-                  <artifactId>hive-storage-api</artifactId>
-                </exclusion>
-                <exclusion>
-                  <groupId>com.esotericsoftware</groupId>
-                  <artifactId>kryo-shaded</artifactId>
-                </exclusion>
-              </exclusions>
-            </dependency>
-            <dependency>
-              <groupId>org.apache.orc</groupId>
-              <artifactId>orc-shims</artifactId>
-              <version>${orc.version}</version>
-              <exclusions>
-                <exclusion>
-                  <groupId>org.apache.hadoop</groupId>
-                  <artifactId>hadoop-common</artifactId>
-                </exclusion>
-                <exclusion>
-                  <groupId>org.apache.hadoop</groupId>
-                  <artifactId>hadoop-hdfs</artifactId>
-                </exclusion>
-              </exclusions>
-            </dependency>
-            <dependency>
-              <groupId>org.apache.hive</groupId>
-              <artifactId>hive-storage-api</artifactId>
-              <version>${hive.storage.api.version}</version>
-            </dependency>
-            <dependency>
-              <groupId>com.google.protobuf</groupId>
-              <artifactId>protobuf-java</artifactId>
-              <version>${protobuf.java.version}</version>
-            </dependency>
-            <dependency>
-              <groupId>com.google.flatbuffers</groupId>
-              <artifactId>flatbuffers-java</artifactId>
-              <version>${flatbuffers.java.version}</version>
-            </dependency>
```

Comment:
Was this deleted accidentally?  The plugin uses flatbuffers directly, and I don't think this is related to ORC.